src/main/java/core/basesyntax/service/FruitItemServiceImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitItem;
+
+public class FruitItemServiceImpl implements FruitItemService {
+
+    @Override
+    public FruitItem create(String fruitName, int quantity) {
+        return new FruitItem(fruitName, quantity);
+    }

```

Comment:
what is the reason for this class? There is only constructor inside the method

Reply:
The reason for this class is creation of FruitItem, as Single Responsibility Principle. 
(Similar to creating an account in Bohdan Chupika's educational video about BankService)